docs/Reference/RadioButton.md

```diff
@@ -66,7 +66,7 @@ export default class Plugin extends React.Component {
 ```
 **Javascript**
 ```
-var radioBtn = new kintoneUIComponent.RadioButton({
+var radioBtn = new kintoneUIComponentComponent.RadioButton({
```

Comment:
@r3-yonashiro 
Please fix this issue, the code is "kintoneUIComponent", not "kintoneUIComponentComponent"